authlogic.gemspec

```diff
@@ -25,9 +25,9 @@ require "authlogic/version"
   s.required_ruby_version = ">= 2.4.0"
 
   # See doc/rails_support_in_authlogic_5.0.md
-  s.add_dependency "activemodel", [">= 5.2", "< 6.1"]
-  s.add_dependency "activerecord", [">= 5.2", "< 6.1"]
-  s.add_dependency "activesupport", [">= 5.2", "< 6.1"]
+  s.add_dependency "activemodel", ">= 5.2"
+  s.add_dependency "activerecord", ">= 5.2"
+  s.add_dependency "activesupport", ">= 5.2"
```

Comment:
```suggestion
  s.add_dependency "activemodel", [">= 5.2", "< 6.2"]
  s.add_dependency "activerecord", [">= 5.2", "< 6.2"]
  s.add_dependency "activesupport", [">= 5.2", "< 6.2"]
```

Reply:
@jaredbeck The compatibility file that paper trail has looks like a good solution! Would it be possible to loosen the constraint until something like the papertrail solution is in place?  It was unclear to me if there was a known issue preventing paperclip from being happy on 6.1 or if the constraint was arbitrary (this is common in the ecosystem). The version constraint also disincentivizes folks from running ahead and testing out compatibility by adding friction (having to fork and edit the gemfile).

Reply:
Good analysis, and good questions :) The constraint exists because we expect Rails to make breaking changes in minor versions.

> Would it be possible to loosen the constraint until something like the papertrail solution is in place?

I'd prefer to wait.